pkg/ddl/sequence.go

```diff
@@ -220,6 +220,16 @@ func alterSequenceOptions(sequenceOptions []*ast.SequenceOption, ident ast.Ident
 			restartValue = op.IntValue
 		}
 	}
+	for _, op := range tableOptions {
+		switch op.Tp {
+		case ast.TableOptionComment:
+			oldSequence.Comment = op.StrValue
+		case ast.TableOptionEngine:
```

Comment:
previously, TiDB implement self sequence according what Mariadb does
```
create sequence seq;
alter sequence seq comment="haha";
ERROR 1064 (42000) at line 3: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'comment="haha"' at line 1
```
And seems Mariadb will error about this, is it reasonable?

Reply:
That's indeed a valid concern.

However I've noticed some more differences between our implementation and the one from MariaDB:
- `SHOW CREATE TABLE <sequence>` has completely different output.

Reply:
PostgreSQL allows updating the comment, but the syntax is completely different.
```
postgres=# CREATE SEQUENCE s1;
CREATE SEQUENCE
postgres=# COMMENT ON SEQUENCE s1 IS 'Example comment';
COMMENT
postgres=# \d+
                                        List of relations
 Schema | Name |   Type   |  Owner   | Persistence | Access method |    Size    |   Description   
--------+------+----------+----------+-------------+---------------+------------+-----------------
 public | s1   | sequence | postgres | permanent   |               | 8192 bytes | Example comment
(1 row)

postgres=# COMMENT ON SEQUENCE s1 IS 'Updated sequence comment';
COMMENT
postgres=# \d+
                                             List of relations
 Schema | Name |   Type   |  Owner   | Persistence | Access method |    Size    |       Description        
--------+------+----------+----------+-------------+---------------+------------+--------------------------
 public | s1   | sequence | postgres | permanent   |               | 8192 bytes | Updated sequence comment
(1 row)

postgres=# SELECT VERSION();
                                                      version                                                      
-------------------------------------------------------------------------------------------------------------------
 PostgreSQL 15.6 (Debian 15.6-0+deb12u1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
(1 row)
```